fuzzers/qemu_arm_launcher/Makefile.toml

```diff
@@ -2,6 +2,7 @@
 [env]
 FUZZER_NAME='libpng_harness'
 PROJECT_DIR = { script = ["pwd"] }
+CROSS_CC = "arm-linux-gnueabi-gcc"
```

Comment:
We could probably try to patch build.rs of libafl_qemu to figure this binary out by itself

Reply:
I agree that hardcoding the CROSS_CC is not a good idea. The problem is that this fuzzer needs a CROSS_CC to function. Figuring out which cross_cc is installed is hard and also systems might have multiple cross_cc installed.

Especially for the github workflows it needs to be set.

Reply:
For the scope of this PR it's more than fine, I was thinking in general

Reply:
You are right, that would be great. I also thought about it, but there are just so many cross_cc. Already for arm on ubuntu using apt you might have arm-none-eabi-gcc, arm-linux-gnueabi-gcc and arm-linux-gnueabihf-gcc installed. Not even speaking about other targets then arm32 or manually installed cross_cc. I think it is impossible to pick the correct one for the user or do you have any idea on how to?